.github/workflows/reusable-workflow.yml

```diff
@@ -102,7 +102,7 @@ jobs:
 
       - name: Upload e2e artifacts as workflow artifacts
         uses: actions/upload-artifact@v4
-        if: ${{ inputs.is_emerynet_test && !cancelled() }}
+        if: ${{ !inputs.is_emerynet_test && !cancelled() }}
```

Comment:
i think this should be reverted

Reply:
No, it should be retained. If tests are not for emerynet, we should retain artifacts.

Reply:
right. my bad